wait-for-postgres.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# wait-for-postgres.sh
+
+set -e
+
+host="$1"
+shift
+cmd="$@"
+
+until PGPASSWORD='postgres' psql -h "$host" -U "postgres" -c '\q'; do
+  >&2 echo "Postgres is unavailable - sleeping"
+  sleep 1
+done
+
+>&2 echo "Postgres is up - executing command"
+exec $cmd
```

Comment:
Since we have a `bin` folder, should we place this script in that folder as well @qjflores or @crankycookie ?

Reply:
It could go in the bin. I don't have a strong opinion about it.

Reply:
moved it to the bin folder